code/modules/integrated_electronics/core/device.dm

```diff
@@ -0,0 +1,84 @@
+/obj/item/device/assembly/electronic_assembly
+	name = "electronic device"
+	desc = "It's a case for building electronics with. It can be attached to other small devices."
+	icon_state = "setup_device"
+	var/opened = 0
+
+	var/obj/item/device/electronic_assembly/device/EA
```

Comment:
please use a better name than EA

Reply:
This file is unchecked now. Because device isn't ported yet. If you don't mind, I'll handle all related to implant and device things later.